docs/source/enterprise-guide/change-management.mdx

```diff
@@ -0,0 +1,468 @@
+    F -.-> C
+    F --> G(Third party API)
+    G -.-> F
+```
+
+The ability to update components independently is a key benefit of federation.
+However, understanding and de-risking changes to a component can be challenging,
+especially as changes may cascade and require changes in other components. This
+is especially true for GraphQL schema defintions, which must be coordinated
+between subgraph services and the gateway.
+
+This section covers the seven most common change management scenarios in a
+federated architecture. We'll consider the proper sequencing of release events,
+how Apollo tooling makes managing changes easier, and when progressive delivery
+techniques like blue-green deploys are useful.
+
+Specifically, we'll examine scenarios that:
+
+**Involve a schema change**
+
+- [Backwards-compatible subgraph schema changes](#backwards-compatible-subgraph-schema-changes)
+- [Backwards-_incompatible_ subgraph schema changes](#backwards-incompatible-subgraph-schema-changes)
+
+**Involve schema changes in multiple subgraphs**
+
+- [Changes to shared types](#changes-to-shared-types)
+- [Changes to query planning](#changes-to-query-planning)
+
+**Don't involve schema changes**
+
+- [Changes to a data source](#changes-to-a-data-source)
+- [Changes to subgraph resolvers](#changes-to-subgraph-resolvers)
+- [Changes to gateway logic/configuration](#changes-to-gateway-logic-or-configuration)
+
+The release guidelines for each scenario should generally apply to all release
+management systems, whether we're deploying to Kubernetes, serverless, or bare
+metal. Some implementation details may change depending on whether we have a
+manual deployment process, a continuous delivery process, or something in
+between.
+
+## Backwards-compatible subgraph schema changes
+
+A schema change is backwards-compatible if it doesn’t affect existing
+operations. Some examples of backwards-compatible changes:
+
+- Adding a type or field — unused until a client includes the fields in new
+  operations.
+- Adding an optional argument to existing fields (ideally with a default value)
+  — the argument will have a null or default value until a client uses it in new
+  operations.
+- Making a nullable field non-nullable — this limits the set of possible output
+  values.
+- Removing or changing _unused_ schema elements.
+
+> This also includes changes to schema metadata like adding or changing a
+> deprecation or description, but these changes aren’t relevant as they don’t
+> affect runtime behavior.
+
+### Release Steps
+
+The release steps for backwards-compatible changes are as follows:
+
+1. In the subgraph code repository, commit a changeset containing a schema
+   change and the corresponding implementation to the release branch, triggering
+   the release pipeline.
+2. The release pipeline performs prerelease steps like running tests.
+3. The release pipeline builds the deployable artifacts:
+   1. The container, jar, or zip of the runtime code.
```

Comment:
Make this sublist an unordered list, because it doesn't describe a sequence of steps